records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+
+Some of our own contributions include
+- [docs.scala-lang#2579][docs.scala-lang#2579], where we migrated the "tour of scala - package objects" page to discuss ways to achieve "top level definitions" in Scala 2/3;
+- [docs.scala-lang#2630][docs.scala-lang#2630] and [docs.scala-lang#2631][docs.scala-lang#2631] to enhance the implicit conversions chapter of the Tour of Scala;
+- [docs.scala-lang#2597][docs.scala-lang#2597], where we added version tabs to the [Migration Tutorial](https://docs.scala-lang.org/scala3/guides/migration/tutorial-intro.html);
+- [docs.scala-lang#2598][docs.scala-lang#2598], [docs.scala-lang#2599][docs.scala-lang#2599], [docs.scala-lang#2602][docs.scala-lang#2602], [docs.scala-lang#2614][docs.scala-lang#2614], and [docs.scala-lang#2617][docs.scala-lang#2617] for updating various chapters of the Scala 3 book.
+
+In addition, we have also made some improvements to usability of the site:
+- [docs.scala-lang#2639][docs.scala-lang#2639] to fix the language dropdown of getting-started guide on translated pages.
+- [docs.scala-lang#2637][docs.scala-lang#2637] to ensure all translations have the same layout for the landing page.
+- [docs.scala-lang#2634][docs.scala-lang#2637] make the "outdated page" notice sticky so that the link to redirect to the updated page is always visible.
+
+### Scala Toolkit
+
+For Scala 2 and Scala 3.
+
+The goal of this project is to help the beginner-to-intermediate-level programmers to learn and use Scala in small projects, such as a script.
+To this end, we will publish a series of written tutorials that focus on common programming tasks, such as testing, reading and writing files, sending HTTP requests, etc.
+
+We started writing a few tutorials for unit-testing using the MUnit library, in [scalacenter/docs.scala-lang#7096](https://github.com/sbt/sbt/pull/7096). The Scala Toolkit series will be published later this year.
+
+### An integrated Scala.js ecosystem
+
+For Scala 2 and Scala 3.
+
+The Scala.js ecosystem contains individual pieces of great quality, from the compiler to the UI libraries. However, it is a challenge for every newcomer to find the pieces that are relevant, to connect them, and to build a good development experience. To address this issue, we want to provide a clear “integrated Scala.js ecosystem”. You can find the complete roadmap [here](https://contributors.scala-lang.org/t/the-scala-centers-roadmap-for-a-unified-scala-js-ecosystem/5568).
+
+The second video in the series, [Getting Started with Scala.js, Laminar and ScalablyTyped](https://www.youtube.com/watch?v=hWUAVrNj65c) was published in December 2022.
+We gave talks on the same topic at Scala IO and ScalaCon.
```

Comment:
```suggestion
We gave talks on the same topic at ScalaIO and ScalaCon.
```

Reply:
Can we link to the talks if they are online?

Reply:
The ScalaCon one is on YouTube, I’ve added a link in 710de3c